modules/order/config/install/profile.type.customer.yml

```diff
@@ -9,3 +9,4 @@ label: Customer
 registration: false
 multiple: true
 weight: 0
+use_revisions: true
```

Comment:
@krystalcode I've made the changes recommended. So, we're not hiding the delete button manually. The ProfileAccessControlHandler() class extends UncacheableEntityAccessControlHandler() and basically, the access to the delete operation is controlled there. So, if we don't give the user permission to delete the customer profile entities, then, it won't display for the user in the operations section and the Profile form as well.